Add usage instructions to README (#12)
shirakaba/expoot: README.md

# About

Best-effort support for using Expo with desktop platforms [react-native-macos](https://github.com/microsoft/react-native-macos) and [react-native-windows](https://github.com/microsoft/react-native-windows).

> # Usage instructions

Reviewer: nit: I think just "Usage" is a bit snappier

```suggestion
# Usage
```

> 

Reviewer: nit: Although a little redundant, let's be clear about what the command does, given we're asking users to blindly run `npx`:

```suggestion

To create a new Expo app that targets iOS, Android, macOS, and Windows, run the following command:

```

```sh

> npx expo-desktop create-app

Reviewer: Let's avoid repeat users ending up with an outdated version:

```suggestion
npx expo-desktop@latest create-app
```

```

> When entering the project directory, you'll have a diff of the changes expo-desktop has made relative to a clean expo app.

Reviewer: Ah, I'd consider that unintentional, actually. Let's just add a step at the end of `create-app` to commit those changes.

```suggestion
```

And until we've implemented that, let's just trust users to work out what to do with the changes themselves (just to avoid overloading with information in our README).

> Note that since prebuild is not yet implemented for desktop, the best course of action is to let `/macos` and `/windows` live in your repo for the time being.

Reviewer: ```suggestion
While `npx expo prebuild` will work as expected for iOS and Android platforms, the equivalent `npx expo-desktop prebuild` command needed for macOS and Windows platforms is not yet implemented, so it is best to commit the `macos` and `windows` directories to source for now. In other words, [CNG](https://docs.expo.dev/workflow/continuous-native-generation/) is not yet supported in Expo Desktop.
```

You can run the macos and Windows versions as per `package.json` scripts just like native mobile:

```sh
bun run macos
bun run windows

> ```

Reviewer: nit: I think we should default to documenting using `npm`, as users of any alternative package manager should be fine to translate.

```suggestion
You can then run your app as follows:

```sh
# Start the Metro bundler
npm run start

# Build and run the iOS, Android, macOS, or Windows targets
npm run ios
npm run android
npm run macos
npm run windows
```
```
